app/controllers/documents/selfies_controller.rb

```diff
@@ -1,19 +0,0 @@
-module Documents
-  class SelfiesController < DocumentUploadQuestionController
-    before_action :set_required_person_names, only: [:edit, :update]
-
-    def self.document_type
-      DocumentTypes::Selfie
-    end
-
-    def after_update_success
-      current_intake.tax_returns.each do |tax_return|
-        tax_return.advance_to(:intake_needs_doc_help)
-      end
-    end
```

Comment:
what is the significance of this `after_update_success`? Do we still need to advance to `intake_needs_doc_help` in the previous controller now?

Reply:
it looks like the [previous controller](https://github.com/codeforamerica/vita-min/blob/main/app/controllers/documents/ids_controller.rb) is already advancing to this status, as is the [next controller](https://github.com/codeforamerica/vita-min/blob/main/app/controllers/documents/ssn_itins_controller.rb)